File: app-ui/src/containers/Main/Recipe.js
import React from "react";

class Recipe extends React.Component {


    render() {
        console.log(this.props.item.description);
        return (
            <div className="recipe">
                <p className="recipe-title">¶ {this.props.item.title}</p>
                <p className="recipe-description"><b style={{fontSize:'15px'}}>Description :</b>{"\n"}{this.props.item.description.replace(/LineBreak/g, '\n')}</p>
                <p className="recipe-ingredients"><b style={{fontSize:'13px'}}>Ingredients :</b>{"\n"}{this.props.item.ingredients.replace(/anItem/g, '\n')}</p>
                <p className="recipe-instructions"><b style={{fontSize:'15px'}}>Instructions :</b>{"\n"}{this.props.item.instructions.replace(/LineBreak/g, '\n')}</p>
            
                <p className="recipe-user"> <hr/>author: {this.props.item.user}</p>
            </div>
        );
    }
}


export default Recipe;

File: app/src/main/java/dao/Recipe.java
package dao;

public class Recipe {

    private String title;
    private String description;
    private String ingredients;
    private String instructions;
    private String user;

    public Recipe(String title, String description, String ingredients, String instructions, String user) {
        this.title = title;
        this.description = description;
        this.ingredients = ingredients;
        this.instructions = instructions;
        this.user = user;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public String getIngredients() {
        return ingredients;
    }

    public void setIngredients(String ingredients) {
        this.ingredients = ingredients;
    }

    public String getInstructions() {
        return instructions;
    }

    public void setInstructions(String instructions) {
        this.instructions = instructions;
    }

    public String getUser() {
        return user;
    }

    public void setUser(String user) {
        this.user = user;
    }

    @Override
    public String toString() {
        return "{\"title\": \"" + title + "\", " +
                "\"description\": \"" + description.replaceAll("\n", "LineBreak") + "\", " +
                "\"ingredients\": \"" + ingredients.replaceAll(",","anItem") + "\", " +
                "\"instructions\": \"" + instructions.replaceAll("\n", "LineBreak") + "\", " +
                "\"user\": \"" + user + "\"}";
    }
}
